introduction.md

```diff
@@ -1,2 +1,11 @@
 it's me, Fauno =)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?